src/components/Thanks/ThanksLayoutV2.vue

```diff
@@ -323,17 +403,26 @@ export default {
 			visibleSection = 'guest';
 		} else if (this.showMgCta) {
 			visibleSection = 'mg';
+		} else if (this.showAutoDepositUpsell) {
+			visibleSection = 'ad';
 		}
 		return {
+			isMobile: false,
 			visibleSection,
 		};
 	},
 	methods: {
+		determineIfMobile() {
+			this.isMobile = document.documentElement.clientWidth < 681;
```

Comment:
Should this be 735, which is our tailwind breakpoint?

Reply:
Had to revert to to match the other layout aspects of this page which still uses foundation. At 735 the content blocks were hidden ;(

Reply:
We'll be able to sync with tailwind when we refactor the layout for Kiva Classic.